src/templates/ApiReference.js

```diff
@@ -187,7 +187,14 @@ const NavItem = ({ isActive, forwardedRef, children, depth }) => {
   }, [isActive, parentDom, isScrollingDown, isMobile]);
 
   return (
-    <NavItemEl isActive={isActive} depth={depth} ref={itemRef}>
+    <NavItemEl
+      isActive={isActive}
+      depth={depth}
+      ref={itemRef}
+      onClick={() => {
+        onNavClick(true);
```

Comment:
This is the only use of `onNavClick` right? The `true` doesn't really have a meaning here, it's an implementation detail of `ScrollRouter`, I think it would be clearer if the function coming out of context were called as `onNavClick()`

Reply:
`onNavClick` is also used in `ScrollRouter` to set it as `false` for `findActiveNode` to use the `isScrollingDown` approach.

Reply:
what if `onNavClick` is called `setIsNavClicked` instead? it'd clear that it's setting a boolean `isNavClicked` in `ScrollRouter`, so passing `true` to it here would make sense

Reply:
oh, i see that it's following the convention of calling it `onNavClick` from `SideNavProgressContext`. 

I think calling it `setIsNavClicked` makes it more clear that this function requires a boolean, but if you don't want to change this convention everywhere, i understand

Reply:
Updated the names! Will merge.